doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1640,18 +1669,46 @@ The view is applied to the top assumption.
   variable, using the |SSR| ``case`` tactic described in
   :ref:`the_defective_tactics_ssr`. It then behaves as the corresponding
   branching :token:`i_pattern`, executing the
-  sequence:token:`i_item`:math:`_i`  in the i-th subgoal generated by the
+  sequence :token:`i_item`:math:`_i`  in the i-th subgoal generated by the
   case analysis; unless we have the trivial destructing :token:`i_pattern`
   ``[]``, the latter should generate exactly m subgoals, i.e., the top
   variable should have an inductive type with exactly m constructors [#7]_.
   While it is good style to use the :token:`i_item` i * to pop the variables
   and assumptions corresponding to each constructor, this is not enforced by
   |SSR|.
+``(`` :token:`i_item` * ``| … |`` :token:`i_item` * ``)``
+  *branching* :token:`i_pattern`. With no exception
+  it executes the sequence :token:`i_item`:math:`_i` on the i-th subgoal.
+  Raises an error if the number of goals and the number of :token:`i_item`
+  mismatch. Note that ``()`` can be used to assert that there is only one
+  goal.
+``[^`` :token:`ident` ``]``
+  *block destructing* :token:`i_pattern`. It performs a case analysis
+  on the top variable and introduces, in one go, all the variables coming
+  from the case analysis. The names of these variables are obtained by
+  taking the names used in the inductive type declaration and prefixing them
+  with :token:`ident`. If the intro pattern immediately follows a call
+  to ``elim`` with a custom eliminator (see :ref:`custom_elim_ssr`) then
+  the names are taken from the ones used in the type of the eliminator.
+``[^~`` :token:`ident` ``]``
+  *block destructing* using :token:`ident` as a suffix.
+``(^`` :token:`ident` ``)``
+  *block introduction* using :token:`ident` as a prefix.
+``(^~`` :token:`ident` ``)``
+  *block introduction* using :token:`ident` as a suffix.
 ``/`` :token:`term`
-  Interprets the top of the stack with the view :token:`term`.
+  interprets the top of the stack with the view :token:`term`.
   It is equivalent to ``move/term``. The optional flag ``{}`` can
   be used to signal that the :token:`term`, when it is a context entry,
   has to be cleared.
+``/ltac:(`` :token:`tactic` ``)``
```

Comment:
According to the grammar this is not an `i_pattern` but an `i_view`. So move its description in the appropriate place, I guess.